File: Python/From1to10.py
#
#   Include the time module, which will be used to access the sleep function
#
import time

#
#   Wait for the user to press any key to start the program
#
a = input("Press Enter to Continue")

#
#   Execute a conditional loop that will run 10 times
#
for i in list(range(1,11)):

    #
    #   Write the current count variable to the output
    #
    print(i);

    #
    #   Pause for one second, before continuing
    #
    time.sleep(1);

#
#   This is the end of the program
#

File: Go/From1to10.go
/*
 *  Wait for the user to press any key to start the program
 */
package main

import (
    "bufio";
    "fmt";
    "os";
    "time";
)

func main() {
    reader := bufio.NewReader(os.Stdin)
    fmt.Print("Press Enter to Continue")
    reader.ReadString('\n')

    /*
     *  Execute a conditional loop that will run 10 times
     */
    for i := 0; i < 10; i++ {
        /*
         *   Write the current count variable to the output
         */
        fmt.Println(i)

        /*
         *  Pause for one second, before continuing
         */
        time.Sleep(1 * time.Second);
    }
}

/*
 *   This is the end of the program
 */
